packages/js/test/plugins/auctionHouseModule/createBid.test.ts

```diff
@@ -208,3 +210,103 @@ test('[auctionHouseModule] create public receipt-less bid but cannot fetch it af
+  // Given we have an Auction House and an NFT.
+  const mx = await metaplex();
+  const seller = await createWallet(mx);
+  const nft = await createNft(mx, {}, { owner: seller.publicKey });
+
+  const auctioneerAuthority = Keypair.generate();
+
+  const { client } = await createAuctionHouse(mx, {
+    auctioneerAuthority,
+  });
+
+  // When we create a private bid on that NFT for 1 SOL.
+  const { bid, buyerTradeState } = await client
+    .bid({
+      mintAccount: nft.mintAddress,
+      seller: seller.publicKey,
+      price: sol(1),
+    })
+    .run();
+
+  // Then we created and returned the new Bid with appropriate defaults.
+  t.equal(bid.tradeStateAddress, buyerTradeState);
+  t.same(bid.price, sol(1));
+  t.same(bid.tokens, token(1));
+  t.false(bid.isPublic);
+
+  // But we cannot retrieve it later with the default operation handler.
+  const promise = client.findBidByAddress(buyerTradeState).run();
+  await assertThrows(
+    t,
+    promise,
+    /The account of type \[BidReceipt\] was not found/
+  );
+});
+
+test('[auctionHouseModule] create public receipt-less Auctioneer bid but cannot fetch it afterwards by default', async (t: Test) => {
+  // Given we have an Auction House and an NFT.
+  const mx = await metaplex();
+  const seller = await createWallet(mx);
+  const nft = await createNft(mx, {}, { owner: seller.publicKey });
+
+  const auctioneerAuthority = Keypair.generate();
+
+  const { client } = await createAuctionHouse(mx, {
+    auctioneerAuthority,
+  });
+
+  // When we create a public bid on that NFT for 1 SOL.
+  const { bid, buyerTradeState } = await client
+    .bid({
+      mintAccount: nft.mintAddress,
+      price: sol(1),
+    })
+    .run();
+
+  // Then we created and returned the new Bid with appropriate defaults.
+  t.equal(bid.tradeStateAddress, buyerTradeState);
+  t.same(bid.price, sol(1));
+  t.same(bid.tokens, token(1));
+  t.ok(bid.isPublic);
+
+  // But we cannot retrieve it later with the default operation handler.
+  const promise = client.findBidByAddress(buyerTradeState).run();
+  await assertThrows(
+    t,
+    promise,
+    /The account of type \[BidReceipt\] was not found/
+  );
+});
+
+test('[auctionHouseModule] create private receipt-less Auctioneer with invalid scope', async (t: Test) => {
+  // Given we have an Auction House and an NFT.
+  const mx = await metaplex();
+  const seller = await createWallet(mx);
+  const nft = await createNft(mx, {}, { owner: seller.publicKey });
+
+  const auctioneerAuthority = Keypair.generate();
+
+  const { client } = await createAuctionHouse(mx, {
+    auctioneerAuthority,
+    auctioneerScopes: [AuthorityScope.Sell],
+  });
+
+  // When we create a private bid on that NFT for 1 SOL.
+  const promise = client
+    .bid({
+      mintAccount: nft.mintAddress,
+      seller: seller.publicKey,
+      price: sol(1),
+    })
+    .run();
+
+  await assertThrows(
+    t,
+    promise,
+    /The Auctioneer does not have the correct scope for this action/
```

Comment:
Interesting... **Out of scope for this PR** but that means we should probably be using Auction House instructions instead of Auctioneer instructions when the scope is not defined in the auctioneer instance. It'll be a pain though because we don't know the scope from the accounts I don't think.

Reply:
It won't allow you to call AH action if hasAuctioneer === true. Instead, it will throw "Must use auctioneer handler."

Reply:
Oh, I see! Ignore me then.